plugin-modernizer-cli/src/main/java/io/jenkins/tools/pluginmodernizer/cli/Main.java

```diff
@@ -66,8 +72,25 @@ public String getVersion() {
         }
     }
 
+    public void listAvailableRecipes() {
+        Yaml yaml = new Yaml();
+        InputStream inputStream = getClass().getClassLoader().getResourceAsStream("recipe_data.yaml");
```

Comment:
Can we move this constant to Settings.java ? I think were are few other place where it's used